index.js

```diff
@@ -66,7 +68,7 @@ if (!isProduction) {
 // production error handler
 // no stacktraces leaked to user
 // eslint-disable-next-line no-unused-vars
-app.use((err, req, res, next) => {
```

Comment:
The `next` you removed is actually used for error handling on the production.

Reply:
Yeah. I was able to figure it out.